saspy/sasutil.py

```diff
@@ -108,13 +105,12 @@ def hpsample(self, **kwargs: dict) -> 'SASresults':
 
         ``required_set = {}``
 
-        ``legal_set= {'cls', 'code', 'grow', 'id', 'model', 'out'
-        'partition', 'performance', 'prune', 'rules'}``
+        ``legal_set= { 'class', 'performance', 'target', 'var'}
```

Comment:
class should be cls, this is the same for outside the documentation for the actual assignment of legal_set